docs/src/operate/registration/http_api.md

```diff
@@ -83,158 +83,79 @@ GET /v1/entities/{topic-id}
 curl http://localhost:8000/te/v1/entities/device/child01
 ```
 
-## Query entities
+## Update entity
+
+An existing entity can be updated using the PATCH API.
+But the updates are limited to the `@parent` and `@health` endpoints only.
+Other properties like `@type` and `@id` can not be updated after the registration.
+
+The `@parent` must be another device, as services cannot have children.
+Similarly, the `@health` endpoint must be a service, as only services publish their health statuses.
+The entities specified in either fields must be registered up-front, before they can be used.
 
 **Endpoint**
 
 ```
-GET /v1/entities
+PATCH /v1/entities/{topic-id}
 ```
 
-**Query parameters**
+**Payload**
 
-| Parameter | Description                                                            | Examples                            |
-|-----------|------------------------------------------------------------------------|-------------------------------------|
-| `root`    | Entity tree starting from the given `root` node (including it) | `device/child2//`                   |
-| `parent`  | Direct child entities of the given `parent` entity (excluding it)             | `device/main//`                     |
-| `type`    | Entities of the given entity `type`                          | `main`, `child-device` or `service` |
+```json
+{
+    "@parent": "{new-parent-topic-id}",
+    "@health": "{some-service-topic-id}"
+}
+```
 
-The following restrictions apply:
-* Multiple values can not be specified for the same parameter.
-* The same parameter can not be repeated multiple times.
-* The `root` and `parent` parameters can not be used together.
+**Example**
 
```

Comment:
A brief sentence explaining what the example is doing would be helpful for users to pick up on user-related purpose of the command quicker.